app.rb

```diff
@@ -250,6 +272,11 @@ def biometric_comparison_required_value(ial)
       ial == 'biometric-comparison-required'
     end
 
+    def requires_enhanced_ipp?(ial)
```

Comment:
I’d expect this method to return `nil` when it returns from line 276.  I know there's [a Ruby style convention ](https://rubystyle.guide/#bool-methods-qmark) that if a method returns a boolean, then the method should end with `?`.     Should a method also end with `?` if it returns a boolean or `nil`?

Reply:
I am now returning false, see commit d9607e4706256f48603e3703c5cf26b5559b0e71